client/blocks/reader-full-post/style.scss

```diff
@@ -482,7 +482,6 @@
 	margin: 0;
 	padding: 0;
 	position: relative;
```

Comment:
@bluefuton We no longer need `position` if we remove `top`.

Reply:
Excellent point @jancavan! I've just removed it.